templates/questions/service/advancedPortHTTP.yaml

```diff
@@ -1,21 +1,14 @@
-                          - variable: advanced
-                            label: Show Advanced Settings
+                          - variable: protocol
+                            label: Port Type
```

Comment:
I don't want this shown by default, that's why it's still under the checkbox...

Reply:
It's either checkbox or completely remove from the questions...

Reply:
All other (https, tcp, udp) are not behind a checkbox..
It was only this one.

Reply:
In that case lets not have it behind a checkbox either :)